generator/src/test/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGeneratorTest.java

```diff
@@ -110,4 +99,28 @@ public void testGeneratesValidExpressions() throws Exception {
     }
   }
 
+  private List<Stmt> makeStmtList(BasicType basicType,
+                                  ShadingLanguageVersion shadingLanguageVersion) {
+    List<Stmt> newStmts = new ArrayList<>();
+    final GenerationParams generationParams = GenerationParams.large(ShaderKind.FRAGMENT, true);
+    for (int i = 0; i < 1000; i++) {
+      final IRandom generator = new RandomWrapper(i);
```

Comment:
^ Please replace 1000 with a named constant, and a brief comment to indicate why we want to loop a bunch of times.

Reply:
Done! use a variable instead of the fixed number and add a comment